package.json

```diff
@@ -21,10 +21,14 @@
     "devDependencies": {
         "browserify": "^13.3.0",
         "clean-css-cli": "^4.0.0",
-        "eslint": "^7.0.0",
+        "eslint": "^7.32.0",
         "katex": "^0.8.0",
         "uglify-js": "^3.6.0",
-        "watchify": "^3.9.0"
+        "watchify": "^3.11.1"
     },
-    "license": "MIT"
+    "license": "MIT",
+    "dependencies": {
+        "clean-css": "^5.2.4",
+        "uglifyjs": "^2.4.11"
+    }
```

Comment:
I think `clean-css` and `uglifyjs` should be `devDependencies` because they are only required during builds, not required to use the generated .js file as a library.

Can we remove this `dependencies` block entirely?

Reply:
I think you're right -- I just moved them per your suggestion in 85288ba64d1fa992155ea0861c563aadb05c7a95

It builds just fine.

Reply:
Great! Thanks.